modules/docs/src/Volo.Docs.Domain/Volo/Docs/Documents/IDocumentRepository.cs

```diff
@@ -12,6 +12,8 @@ public interface IDocumentRepository : IBasicRepository<Document>
         Task<List<DocumentInfo>> GetUniqueListDocumentInfoAsync(CancellationToken cancellationToken = default);
 
         Task<List<Document>> GetListByProjectId(Guid projectId, CancellationToken cancellationToken = default);
+        
+        Task ClearCachesAsync(Guid projectId, CancellationToken cancellationToken = default);
```

Comment:
`ClearCache` is like a business method. The `Repository` should have no business logic. 
Consider: `UpdateProjectLastCachedTime(Guid projected, DateTime? cachedTime)`


We can make the `LastCachedTime` of `Entity` nullable if necessary.